data/json/recipes/recipe_others.json

```diff
@@ -3690,7 +3690,7 @@
     "tools": [ [ [ "surface_heat", 20, "LIST" ] ] ],
     "components": [
       [ [ "water_clean", 2 ], [ "water", 2 ] ],
-      [ [ "tanbark", 1 ], [ "acorns", 2 ], [ "hops", 2 ], [ "pine_bough", 6 ] ],
+      [ [ "tanbark", 1 ], [ "acorns", 2 ], [ "hops", 2 ], [ "pine_bough", 6 ], [ "brain", 3 ] ],
       [ [ "salt_water", 2 ], [ "saline", 10 ], [ "salt", 10 ] ],
       [ [ "edible_tallow_lard", 1, "LIST" ], [ "tallow_tainted", 1 ] ],
```

Comment:
I'm not an expert in tanning, but from my understanding of the [source in the original issue](http://livetheoldway.com/brain-tanning-deer-hides/), I assume the brain is working like tallow, rather than as a tanning agent?

Quote from the source:
> By soaking the hide in lye, you will be removing the hair and penetrating the skin layers to clean out the mucus in between them so that in later steps, the oil from the dressing (brains) can completely penetrate the hide.

Reply:
According to: https://animals.howstuffworks.com/mammals/deer-tan-own-hide1.htm
The brain contains lecithin, an oil which works as a tanning agent. 

> The brain contains oil called lecithin that serves as a natural tanning agent to lubricate the skin. Native Americans first practiced this method of brain tanning and continue to tan hides with it today. Ironically, deer have just enough oil, or lecithin, in their brains to adequately tan their own hides. The same goes for other animals including cows, raccoons and beavers. Once the hide has absorbed the liquid, the skin is smoked in order to seal it.

> For those who aren't interested in getting inside an animal's head, you can also obtain tannin from plant sources, such as sumac leaves or bark. 

From this description it is working a similar function to the acorns, tanbark etc, which is why I chose to add it as an option on that line instead of the tallow line. I initially thought the same as you though.